### PR TITLE
chore: use decentraland-dapps transaction module

### PR DESCRIPTION
**Pull request by cazala**
This PR bumps decentraland-eth and decentraland-dapps and replaced the transaction module with the one in decentraland-dapps

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -61,6 +62,7 @@
     "query-string": "^5.1.0",
     "raf": "3.4.0",
     "react": "16.2.0",
+    "react-circular-progressbar": "^1.0.0",
```
Comment (NicoSantangelo) [inline webapp/package.json]: I don't see where this is used?

**Comment by NicoSantangelo** [comment]
image
clarification on the error

File: webapp/package.json (hunk 1 above)
Comment (cazala) [reply webapp/package.json]: uh this is going to be used by the new UI (for the confirmations progressbar)

**Comment by cazala** [comment]
@NicoSantangelo fixed by ea88e7dea397a67d096fbe2f697de6045a9ae272

**Pull request merged**